extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -714,9 +711,28 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 		return this._pythonExecutable;
 	}
 
-	public getCondaExePath(): string {
-		return path.join(this._pythonInstallationPath,
-			process.platform === constants.winPlatform ? 'Scripts\\conda.exe' : 'bin/conda');
+	public get condaExecutable(): string | undefined {
+		return this._condaExecutable;
+	}
+
+	public static async getCondaExePath(pythonInstallationPath: string): Promise<string> {
+		let exeName = process.platform === constants.winPlatform ? 'Scripts\\conda.exe' : 'bin/conda';
+		let condaPath = path.join(pythonInstallationPath, exeName);
+		let condaExists = await fs.pathExists(condaPath);
+		// If conda was not found, then check if we're using a virtual environment
+		if (!condaExists) {
+			let pathParts = pythonInstallationPath.split(path.sep);
+			if (pathParts.length > 1 && pathParts[pathParts.length - 2] === 'envs') {
+				// The root Anaconda folder is 2 folders above the virtual environment's folder
```

Comment:
Is this documented somewhere? If so would be good to add a link to that

Reply:
Added a link to the docs